### PR TITLE
[DOC] Add instructions for download R dependencies

### PR DESCRIPTION
**Pull request by j143**
.

### CODE REVIEW

**Comment by phaniarnab** [comment]
Why an user needs to install R?

File: docs/site/install.md [hunk 1]
```diff
@@ -83,6 +83,13 @@ sudo add-apt-repository 'deb https://cloud.r-project.org/bin/linux/ubuntu focal-
 sudo apt install r-base
 ```
 
+Additionally, you need to install the R depedencies for integration tests, like this:
```
Comment (j143) [inline docs/site/install.md:86]:
```suggestion
Optionally, you need to install the R depedencies for integration tests, like this:
```

File: docs/site/install.md [hunk 2]
```diff
@@ -127,6 +134,13 @@ export JAVA_HOME=`/usr/libexec/java_home -v 1.8`
 
 For running all tests [r-base](https://cran.r-project.org/bin/macosx/) has to be installed as well since this is used as a secondary system to verify the correctness of our code, but it is not a requirement to enable building the project.
 
+Additionally, you need to install the R depedencies for integration tests, like this:
```
Comment (j143) [inline docs/site/install.md:137]:
```suggestion
Optionally, you need to install the R depedencies for integration tests, like this:
```

**Comment by j143** [comment]
> Why an user needs to install R?

This doc is also, helpful for developers. (intent of this is for systemds developers too!)

**Comment by phaniarnab** [comment]
If I am not wrong, the current doc doesn't tell how to build the test classes. And that is because the primary target is the users who want to build SystemDS from the source.

**Comment by Baunsgaard** [comment]
> If I am not wrong, the current doc doesn't tell how to build the test classes. And that is because the primary target is the users who want to build SystemDS from the source.

I agree with this statement, for the devs we should do an extended guide.

**Pull request closed without merging**